lang/en.js

```diff
@@ -69,6 +69,8 @@ export default {
 	"components.input-date.errorOutsideRange": "Date must be between {minDate} and {maxDate}",
 	"components.input-date.openInstructions": "Use date format {format}. Arrow down or press enter to access mini-calendar.",
 	"components.input-date.now": "Now",
+	"components.input-date.revertWithFormat": "{label} reverted to previous value. Use date format {format}.",
+	"components.input-date.revertWithInstructions": "{label} reverted to previous value. Use date format {format}. Arrow down or press enter to access mini-calendar.",
```

Comment:
It doesn't reverse them in Arabic, either because there are people smarter than me who make sure these things work or because magic.

Reply:
Interesting, so it sounds like this could be bad in Asian languages then? Would you mind checking out what I've ended up doing? I basically have 3 separate lang terms that I combine in a few different ways depending on the situation. 

For the new functionality we just want the "[field] was reverted" on a separate line. If needed, we could go back to what's in this previous commit with them pre-combined for the second paragraph content.

Reply:
Yeah I'll check a few more things today. This will be good to understand in general.

Reply:
Ok, after a bunch iterations of ways to render concatenated strings, none of them break Arabic, but none of them don't "break" Japanese and Chinese (we get the unnecessary space between sentences). That seems minor, but I still lean toward not concatenating since these segmentation rules could still be different for future supported languages.

Just waiting on a response here to see if there's any cost downside.

Also, [this](https://learn.microsoft.com/en-us/globalization/internationalization/concatenation) whole thing is good, but a few key blurbs:
>While ending a sentence with a period is common in languages that use the Latin script, other languages use alternatives. For example, Japanese uses 。 (U+3002 IDEOGRAPHIC FULL STOP) while Thai doesn’t consistently use a punctuation mark to separate sentences.

>In English, comparable to word separation, sentences are also separated with spaces. You shouldn't assume that this is true for all scripts and languages.

(The IDEOGRAPHIC FULL STOP has white space built into it, so a separate space isn't used)